### DISCUSSION TITLE
Saleor 3.9: Tax Improvements

### DISCUSSION THREAD

**Discussion by karolkielecki** [saleor/saleor#11433] [Announcements]
Welcome to Saleor 3.9! 🚀

Saleor now offers granular configuration of tax classes per channel and country. This feature enables you to mix dynamic tax calculations done by apps or create static rules. 

Learn more about tax improvements in [the 3.9 changelog](https://github.com/saleor/saleor/releases/tag/3.9.0).

**Comment by kirubeltadesse1** [comment]
Good

**Comment by kendrice** [comment]
Use version 3.9.
Does the storefront need to change?

**Reply by timuric** [reply]
@kendrice there shuld be no breaking changes since 3.5
